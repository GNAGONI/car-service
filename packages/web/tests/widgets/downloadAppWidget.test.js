import React from 'react'
import renderer from 'react-test-renderer'
import DownloadAppWidget from '../../components/widgets/downloadAppWidget'

it('renders correctly', () => {
  const tree = renderer.create(<DownloadAppWidget />).toJSON()
  expect(tree).toMatchSnapshot()
})
