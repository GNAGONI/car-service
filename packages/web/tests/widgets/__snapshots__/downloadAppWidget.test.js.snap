// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="widget app-widget app-downloads"
>
  <div
    className="widget-head text-center"
  >
    <h3>
      Download the car.service app
    </h3>
  </div>
  <ul
    className="app-downloads"
  >
    <li>
      <a
        href="#"
      >
        <img
          alt="Download on the App Store"
          height="29"
          src=""
          width="88"
        />
      </a>
    </li>
    <li>
      <a
        href="#"
      >
        <img
          alt="GET IT ON Google Play"
          height="30"
          src=""
          width="99"
        />
      </a>
    </li>
  </ul>
  <div
    className="img-mobile-holder"
  >
    <img
      alt="image description"
      src=""
    />
  </div>
</section>
`;
